pkg/newRelic/newrelic.go

```diff
@@ -0,0 +1,42 @@
+package newRelic
+
+import (
+	"github.com/newrelic/go-agent/v3/newrelic"
+)
+
+func NewTransaction(nr *newrelic.Transaction) *NewRelicTransaction {
+	return &NewRelicTransaction{
+		nr: nr,
+	}
+}
+
+type NewRelicTransaction struct {
```

Comment:
In golang we don't use prefix in types inside the package. Just use Transaction for this,.
```
type Transaction struct {
	nr *newrelic.Transaction
}
```